package.json

```diff
@@ -38,11 +38,10 @@
     "graphite-udp": "^1.0.0",
     "graphql": "0.11.7",
     "graphql-tag": "2.5.0",
-    "jsdom": "^7.2.2",
+    "jsdom": "^14.0.0",
     "lru-cache": "^4.0.0",
     "node-fetch": "1.7.3",
     "nodemon": "^1.18.7",
-    "raf": "^3.4.0",
```

Comment:
Having jsdom pretend to be visual provides this as far as I can tell, so I don't think we need to do this polyfill anymore. If we do, I think the code we call should be responsible for it, rather than the server.